src/ensembl/src/content/app/browser/browser-bar/BrowserBar.tsx

```diff
@@ -223,15 +222,12 @@ export const BrowserNavigatorButton = (props: BrowserNavigatorButtonProps) => (
 
 const mapStateToProps = (state: RootState): StateProps => ({
   activeGenomeId: getBrowserActiveGenomeId(state),
-  activeObjectId: getBrowserActiveEnsObjectId(state)[
-    getBrowserActiveGenomeId(state)
-  ],
   browserActivated: getBrowserActivated(state),
   browserNavOpened: getBrowserNavOpened(state),
   chrLocation: getChrLocation(state),
   defaultChrLocation: getDefaultChrLocation(state),
   drawerOpened: getDrawerOpened(state),
-  ensObjectInfo: getEnsObjectInfo(state),
+  ensObject: getBrowserActiveEnsObject(state),
```

Comment:
Would it be better to call this as `activeEnsObject` like we have `activeGenomeId`?

Reply:
🤷‍♂  I don't really have any preferences regarding this.

Reply:
We could either rename `activeGenomeId` to just `genomeId` or `ensObject` to `activeEnsObject` to keep them consistent. 
I think using the word `active` makes it more clear?

Reply:
Turns out this `ensObject` is used in multiple places and, what's worse, is passed in other components, which also call it ensObject. I'd much rather leave its name as it is for now. As well as the names of `activeGenomeId` and `activeEnsObjectId`.